### PR TITLE
[redis-ha]Terminate TLS on HAproxy

### PR DESCRIPTION
**Pull request by GGre4**
#### What this PR does / why we need it:

To terminate TLS on HAProxy instead of terminating TLS on redis.

#### Which issue this PR fixes
Currently the chart does not support terminating TLS on haproxy, additionally the chart does not support using redis with both TLS enabled and TLS disabled. To achieve this modifying the haproxy service is needed to have more control on ports.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ X] Chart Version bumped
- [ X] Variables are documented in the README.md
- [ X] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: charts/redis-ha/README.md [hunk 1]
```diff
@@ -150,8 +150,9 @@ The following table lists the configurable parameters of the Redis chart and the
...
-| `haproxy.image.repository`| HAProxy Image Repository                                                                                                                                                                                 | `haproxy`                                                                                  |
+| `haproxy.replicas`        | Number of HAProxy instances                                                                                                                                                                              | `3`
+| `haproxy.image.repository`| HAProxy Image Repository                                                                                                                                                                                 | `haproxy`
+| `haproxy.image.rserviceAccountName`| HAProxy serviceAccountName                                                                                                                                                                                 | `default`                                                                                   |
```
Comment (DandyDeveloper) [inline charts/redis-ha/README.md:155]: `rserviceAccountName` Typo?

**Review by DandyDeveloper — CHANGES_REQUESTED** [review]
@GGre4 Looks good, will test. Have a number of PRs to work through, can you just fix the spelling error?

**Comment by GGre4** [comment]
Typo has been updated

**Comment by GGre4** [comment]
@DandyDeveloper  Any reason why this PR was not approved ? i had updated it to reflect with the updated chart version and such but it was not reviewed

**Comment by DandyDeveloper** [comment]
@GGre4 Can you reopen? I'm really not sure, it just flew under my radar. I'm sorry.

**Pull request closed without merging**